build.gradle

```diff
@@ -169,7 +169,7 @@ def versions = [
         serenity                        : '2.0.23',
         serenityreporter                : '2.1.4',
         pact_version                    : '3.5.24',
-        probateCommons                  : '1.2.0',
+        probateCommons                  : '1.2.1-DTSPB-1526-2',
```

Comment:
hi Nigel, could you please merge the changes onto Probate commons and then create a new tag without Jira id . 
We should have something like 1.2.2
rather than what we have in there currently

Reply:
Done